components/TextFields/examples/TextFieldExample.swift

```diff
@@ -153,6 +153,7 @@ final class TextFieldSwiftExample: UIViewController {
     scrollView.addSubview(message)
     let messageController = MDCTextInputControllerDefault(textInput: message)
     message.textView?.delegate = self
+    message.text = "This is where you could put a multi-line message like an email."
```

Comment:
Can we demo something with line breaks like `\r\n`?

Reply:
Done!

Reply:
DOne